checkstyle-suppressions.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<!DOCTYPE suppressions PUBLIC
+        "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+        "https://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+    <suppress files=".*.java"
+              checks="RegexpSinglelineJava"/>
+</suppressions>
```

Comment:
Please add one more empty line at the end of file.

Reply:
updated